Resources/Prototypes/Maps/europa.yml

```diff
@@ -0,0 +1,67 @@
+- type: gameMap
+  id: Europa
+  mapName: 'Europa'
+  mapPath: /Maps/europa.yml
+  minPlayers: 0
+  maxPlayers: 40
+  stations:
+    Core:
```

Comment:
Europa

Reply:
Silly me, i copied over the thing from core and forgot to rename it